osu.Game.Rulesets.Osu/OsuRuleset.cs

```diff
@@ -143,7 +141,47 @@ public override IEnumerable<Mod> GetModsFor(ModType type)
                         },
                         new OsuModTarget(),
                     };
-
+                case ModType.DifficultyCalculation:
+                    return new Mod[]
+                    {
+                        new MultiMod(),
```

Comment:
what is this?

Reply:
This is supposed to be equivalent to no mod, alternative being we use nulls which sounds wrong, or using an `EmptyMod` type.

Reply:
Definitely worth making a `NoModMod` to disambiguate